src/adapter/src/coord/ddl.rs

```diff
@@ -669,6 +675,11 @@ impl Coordinator {
             .expect("unable to drop temporary items for conn_id");
     }
 
+    fn update_caching_config(&mut self) {
```

Comment:
nit: this is specific to secrets caching rather than being about caching generally, so I feel like we should say so in the method name!

```suggestion
    fn update_secrets_caching_config(&mut self) {
```

Reply:
Updated!